src/main/java/TicTacToe/Session.java

```diff
@@ -0,0 +1,41 @@
+package TicTacToe;
+
+class Session {
+    private static final int MAX_MARK_LENGTH = 2;
+    private Game game;
+    private Display display;
+
+    Session(Game game, Display display) {
+        this.game = game;
+        this.display = display;
+    }
+
+    void start() {
+        display.printGreeting();
+        customiseMark(game.playerX);
+        customiseMark(game.playerO);
+        game.play();
+    }
+
+    private void customiseMark(IPlayer player) {
+        display.promptPlayerToCustomiseMark(player.getMark());
+
+        String markChoice = "";
+        while(!isValid(markChoice)) {
+            markChoice = sanitize(player.getMarkChoice());
+            if(isValid(markChoice)) {
+                player.setMark(markChoice);
+                break;
+            }
+            display.warnInvalidInput(markChoice);
+        }
+    }
+
+    private boolean isValid(String input) {
```

Comment:
Similar to our previous discussion it seems that verifying and sanitizing input in this instance is more of a `display` concern, no? The `session` requests a mark choice from the display. Ideally a valid one already.